### ISSUE TITLE
a too large file upload results in cryptic error message

### ISSUE THREAD

**Issue by spukkin** [redmatrix/hubzilla#74]
When uploading a file that exceeds the limits set in php.ini (which include filesize and time duration of upload), user receives this error message:

<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
<s:exception>Sabre\DAV\Exception\NotImplemented/s:exception
<s:message>There was no handler found for this "POST" method/s:message
<s:sabredav-version>1.8.10/s:sabredav-version
/d:error

a simpler error message such as "file exceeds limits set in php.ini, contact system admin"...or the like would be helpful.

**Comment by kenrestivo** [comment]
code-escaped version of the error is:

```xml
<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
<s:exception>Sabre\DAV\Exception\NotImplemented</s:exception>
<s:message>There was no handler found for this "POST" method</s:message>
<s:sabredav-version>1.8.10</s:sabredav-version>
</d:error>
```

**Comment by dawnbreak** [comment]
I had prepared client side validations against these limits before even transferring anything to the server and also catching more of such exceptions. But it is still in a branch for redmatrix. Unfortunately had no time yet, looking at porting it to hubzilla. If someone feels to help please take a look here: https://github.com/dawnbreak/redmatrix/commits/RedDAV my time was a bit limited in the last weeks.

**Comment by zotlabs** [comment]
Fixed in 2.9 and higher
